locations/spiders/tanger_outlet.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+class TangerOutletSpider(scrapy.Spider):
+    name = "tanger_outlet"
+    allowed_domains = ["www.tangeroutlet.com"]
+    download_delay = 0.1
+    start_urls = (
+        "https://www.tangeroutlet.com/locations",
+    )
+
+    def parse(self, response):
+        storeurls = response.xpath('.//a[@title="Visit"]/@href').extract()
+        storeurls = [response.urljoin(i) for i in storeurls if 'location' in i]
+        for storeurl in storeurls:
+            yield scrapy.Request(storeurl, callback=self.parse_store)
+
+    def parse_store(self, response):
+        tempurl = '/' + response.url.split('/')[-2] + '/'
+        for i in response.xpath('.//ul[@id="usCenters"]/li[@class="allCenters"]/a/@href').extract():
+            if i == tempurl:
+                latitude = float(response.xpath('.//ul[@id="usCenters"]/li[@class="allCenters"]/@data-latitude').extract_first())
+                longitude = float(response.xpath('.//ul[@id="usCenters"]/li[@class="allCenters"]/@data-longitude').extract_first())
```

Comment:
Looks like all the locations are sitting at the lat/lon of one store:

https://placescraper-results.s3.amazonaws.com/map.html?show=https://placescraper-results.s3.amazonaws.com/ci/284742598/tanger_outlet/output.geojson

Looking at the source for https://www.tangeroutlet.com/locations, it seems like you can get all the location information on that single page without having to make subsequent requests. Try putting the for loop that's in your current `parse_store()` function in the `parse()` method and see what happens.

Reply:
Ah, I completely didn't notice that all locations were exactly the same. I too thought of scraping everything from the main page too, with a couple different ways to select the divs . 

For some reason, whenever I targeted the `center-info-block` class it kept returning an empty list. I'll give it another shot and use a different combination of selectors :)